### DISCUSSION TITLE
Did className change?

### DISCUSSION THREAD

**Discussion by niloc7** [KingSora/OverlayScrollbars#472] [Q&A] [answered]
I was wondering if there was some update that wasn't included in the documentation?

I've tried the simple themes, and added this:

    OverlayScrollbars(document.querySelector("body"), {
        className: "os-theme-thin-dark"}

But nothing happens.. and when I "view source" I don't see the os-theme-thin-dark anywhere at all. 

All I see is this same default class:
`<div class="os-scrollbar os-scrollbar-horizontal os-theme-dark os-scrollbar-handle-interactive os-scrollbar-cornerless os-scrollbar-auto-hidden"><div class="os-scrollbar-track"><div class="os-scrollbar-handle" style="width: 90.323%; transform: translateX(0%);"></div></div></div>`

Thanks for your guidance! =)

**Comment by KingSora** [comment]
@niloc7 Good day :)

Which version are you using?

**Comment by niloc7** [comment, marked as answer]
Hey KingSora! 

I'm using the latest version pulled directly from NPM. :)

**Reply by niloc7** [reply]
The full code if you need to see it. Oh btw, the clickScrolling in the docs is outdated too.. it's clickScroll now right?

```document.addEventListener("DOMContentLoaded", function () {
    OverlayScrollbars(document.querySelector("body"), {
        className: "os-theme-thin-dark",
        overflow: {
            x: "hidden",
            y: "scroll",
        },
        scrollbars : {
            clickScroll: true,
        },
    });
});

**Reply by KingSora** [reply]
So in case you are using any of the `v2` beta versions: yes, the `className` option is gone and was replaced with the `scrollbars.theme` option. I also haven't ported any of the `v1` styles besides the default theme to `v2`

**Reply by niloc7** [reply]
:O Oh I was using a beta version?? Ahhh my bad! No wonder it didn't work. 

Hmm.. do you recommend I continue using the beta, and use scrollbars.theme, or should I install v1 instead?

**Reply by KingSora** [reply]
I recommend you using `v2` even if it is in beta. I'll release the final version 2 soon and it is stable enough anyways :)

**Reply by niloc7** [reply]
Alright! :D Thanks KingSora!

One last thing, erm.. how do you put scrollbars.theme.. I'm blind or something.. I really can't seem to find it in any docs at all.. 

And do they still use the same css class names as the "Theme Downloads" on your site?

**Reply by KingSora** [reply]
You can have a look at the [changelog](https://github.com/KingSora/OverlayScrollbars/blob/master/packages/overlayscrollbars/CHANGELOG.md) what exactly changed and how to port things
